README.md

```diff
@@ -82,6 +103,13 @@ The example app is a catalogue of the core components provided by this library.
    4. `CLUSTER_PORT` defaults to `443` unless configured otherwise
    5. `CLUSTER_HOST` the base URL of your CP4D cluster (ex. `example.com`)
 
+    #### Windows Only
+    On Windows, the default `SASS_PATH` environmnt variable must be updated. Append the following to the `.env.local` file:
```

Comment:
I think this (combined with the override in the setup script) is a good enough solution for setting up `SASS_PATH` across multiple environments